mgmake/generate.go

```diff
@@ -76,7 +76,7 @@ endif{{end}}
 			return err
 		}
 	}
-	err = os.WriteFile(makefile, createMakefileVariablesFromMap(mgMakefiles), 0o644)
+	err = os.WriteFile(makefile, createMakefileVariablesFromMap(mgMakefiles), 0o600)
```

Comment:
I guess in a way this 0o600 could be one global constant maybe with text on why its exactly 0o600.

Reply:
It's `gosec` that requires 0600 as permissions for newly created files - IMO octal permissions should be self-explaining (`rw--` in this case)

Reply:
> It's `gosec` that requires 0600 as permissions for newly created files - IMO octal permissions should be self-explaining (`rw--` in this case)
+1